documentation/modules/auxiliary/server/socks4a.md

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Hmm... was this comment originally left on line 1? Has Github gone a bit wrong? The diff now shows this comment against line 3... I think I've addressed this.